[Bug] Can't run the tests of UpdateMethod

Hello, I want to test UpdateMethod in workflow. But I get an error:

```
AssertionError: assert($result !== null)
/var/www/html/vendor/temporal/sdk/src/Client/Update/UpdateHandle.php:128
/var/www/html/vendor/temporal/sdk/src/Client/Update/UpdateHandle.php:89
/var/www/html/vendor/temporal/sdk/src/Client/Update/UpdateHandle.php:75
/var/www/html/vendor/temporal/sdk/src/Internal/Client/WorkflowProxy.php:112
/var/www/html/tests/Workflows/UploadFileWorkflowTest.php:83
```

At the same time, the signal and jquery tests are working fine.

Here is the test code:

```php
public function testCanSaveFileData(): void
    {
        $this->testService->lockTimeSkipping();

        $arguments = [
            'path' => $this->faker()->filePath(),
            'original_name' => $this->faker()->word(),
            'extension' => $this->faker()->fileExtension(),
            'size' => $this->faker()->numberBetween(),
            'is_image' => $this->faker()->boolean(),
        ];

        $file = File::factory()->make($arguments);

        $workflow = $this->workflowClient->newWorkflowStub(UploadFileWorkflow::class);
        $run = $this->workflowClient->start($workflow);

        $this->assertEquals(WorkflowExecutionStatus::Running, $run->describe()->info->status);

        $result = $workflow->saveTempFileData(new SaveFileDataArguments($arguments));

        $this->assertEquals($file->id, $run->describe()->info->searchAttributes->getValue('FileId'));
        $this->assertEquals($file->toArray(), $result['data']);

        $this->testService->unlockTimeSkipping();
    }
```

I tried calling UpdateMethod via `update` and `startUpdate'. the result is exactly the same

--- a/src/Internal/Client/WorkflowStub.php
+++ b/src/Internal/Client/WorkflowStub.php
@@ -330,7 +330,7 @@ final class WorkflowStub implements WorkflowStubInterface, HeaderCarrier
             arguments: EncodedValues::fromValues($args, $this->converter),
             header: Header::empty(),
             waitPolicy: $nameOrOptions->waitPolicy,
-            updateId: $nameOrOptions->updateId ?? '',
+            updateId: $nameOrOptions->updateId ?? Uuid::v4(),
             firstExecutionRunId: $nameOrOptions->firstExecutionRunId ?? '',
             resultType: $nameOrOptions->resultType,
         ));

--- a/tests/Functional/Client/UpdateOnTimeSkippingServerTestCase.php
+++ b/tests/Functional/Client/UpdateOnTimeSkippingServerTestCase.php
@@ -1,0 +1,41 @@
+<?php
+
+/**
+ * This file is part of Temporal package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Temporal\Tests\Functional\Client;
+
+use Temporal\Testing\WithoutTimeSkipping;
+use Temporal\Tests\Workflow\UpdateWorkflow;
+
+/**
+ * @group functional
+ * @group client
+ */
+class UpdateOnTimeSkippingServerTestCase extends AbstractClient
+{
+    use WithoutTimeSkipping;
+
+    public function testUpdateResolvesOnTimeSkippingServer(): void
+    {
+        $client = $this->createClient();
+        $workflow = $client->newWorkflowStub(UpdateWorkflow::class);
+
+        $client->start($workflow);
+
+        /** @see UpdateWorkflow::addNameWithoutValidation */
+        $result = $workflow->__getUntypedStub()
+            ->startUpdate('addNameWithoutValidation', 'Test')
+            ->getResult(5);
+
+        self::assertSame('Hello, Test!', $result);
+
+        $workflow->exit();
+    }
+}

